openmetrics/datadog_checks/openmetrics/data/conf.yaml.example

```diff
@@ -76,6 +76,13 @@ instances:
     #
     # send_histograms_buckets: true
 
+    ## @param send_distribution_buckets - boolean - optional - default: false
+    ## Set `send_histograms_buckets` to `true` to send histograms as Datadog distribution metrics.
```

Comment:
@mfpierre this should not be `send_histograms_buckets` but `send_distribution_buckets`

Reply:
Indeed! Thanks for spotting this, let me correct this 🤦‍♂

Reply:
fixed in #4734